gateway-release-common/home/bin/knox-functions.sh

```diff
@@ -181,7 +218,11 @@ function appStart {
 
       rm -f "$APP_PID_FILE"
 
-      nohup "$JAVA" "$APP_JAVA_OPTS" -jar "$APP_JAR" "$@" >>"$APP_OUT_FILE" 2>>"$APP_ERR_FILE" & printf %s $!>"$APP_PID_FILE" || exit 1
+      if [ ${#APP_JAVA_OPTS[@]} -eq 0 ]; then  #there is no APP_JAVA_OPTS set
```

Comment:
@smolnar82 - do we need this check? `"${APP_JAVA_OPTS[@]}"` should be ok if there are no values set? Should just be an empty string?

Reply:
It's working...I just wanted to emphasize the fact that there are no additional Java opts applied in this case. I changed it anyway; please review.